src/components/Camera/measure_blur.js

```diff
@@ -0,0 +1,98 @@
+//ref: https://github.com/timotgl/inspector-bokeh/blob/master/measure_blur.js
```

Comment:
because it worked, it was lean and it was packaged as one function that outputted a blur score. All other solutions involved me doing extra bit of work.